codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttConnectPayload.java

```diff
@@ -51,6 +65,10 @@ public String willTopic() {
     }
 
     public String willMessage() {
```

Comment:
please deprecate this method an pointing folks to `willMessageBytes()`.
also consider adding to the javadocs that this method may create a new string on each invocation.